.github/workflows/release.yml

```diff
@@ -138,13 +145,18 @@ jobs:
             rustup target add ${{ matrix.cargo_targets }}
     
         - name: build release
+          if: matrix.os != 'windows-latest'
           shell: bash
           run: |
             if [ "${{ matrix.name }}" = "ubuntu-arm-latest" ]; then
               export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_LINKER=aarch64-linux-gnu-gcc
               export CC=aarch64-linux-gnu-gcc
             fi
             cargo build --profile optimized-release --all-features --target ${{ matrix.cargo_targets }}
+        
+        - name: build release
+          if: matrix.os == 'windows-latest'
+          run: cargo build --profile optimized-release --all-features --target ${{ matrix.cargo_targets }}
```

Comment:
I don't get why windows gets its own section here, the cargo command is exactly the same as the unix version above?